### DISCUSSION TITLE
Standard VHF mount adapter for MAIANA

### DISCUSSION THREAD

**Discussion by peterantypas** [peterantypas/maiana#131] [Show and tell]
This is the third time I'm attempting this, and maybe it's going to be the charm.

After spending a couple of months on-and-off getting this to work with the more compact wireless MAIANA (which I since canceled), I have a better idea now about getting an adapter to work with the full size unit.

The basic concept is this:

![maiana-mount1](https://user-images.githubusercontent.com/1565933/236914488-a6a695cf-9e2e-4d20-89f4-0cb36e53b55d.jpg)

Another view here:

![maiana-mount2](https://user-images.githubusercontent.com/1565933/236917109-59c21313-72f4-4959-a19b-4c4bb891b22b.jpg)

This adapter will be made of Nylon (PA12). It will need to be glued onto the main enclosure using a two-part epoxy. 

The slot will allow the RJ45 connector to project through it. It will also support the edges of the PCB, relieving downward pressure on the SMA connector from the weight of the Ethernet cable.

The 1-14 thread in the lower section will fit any standard mount, as long as it can pass the Cat5 cable. Some mounts already have a cutout through their threads for a coax.

One caveat so far is that the enclosure will have to be trimmed in length from the 12" that ships now. This will lower the center of gravity so it's probably a good thing.

Any opinions welcome.

**Comment by sailoog** [comment]
Definitive fixings such as glue are not a good thing, how about a screw or some solution that also strangles or grips the cable?

**Comment by SirReal-surreal** [comment]
I love this idea because here in Germany we do have different diameters e.g. for fiberglass pipes but the 1" OD antenna thread is the same.
So, we get a lot of mountings e.g. for the rail or mast. If it could be screwed, this would be perfect.
If the adapter needs to be glued in with epoxy - no worries. Every sailor has this available and if I get this part doesn't need to be removed.

**Comment by dkroeske** [comment]
Hi, I designed and 3D-print a Maiana holder and cable train relief that fits in the white tube (I use the 1meter NL version https://www.hornbach.nl/p/martens-buis-pvc-o-32-mm-lengte-100-cm/6094263/). The 2 parts are connected using 'slide and fix'. 
![Maiana_fix](https://github.com/peterantypas/maiana/assets/2204527/c631ae0f-9923-4dcb-9dc3-db2a65d815b5)
![IMG_9518](https://github.com/peterantypas/maiana/assets/2204527/ea76b516-a493-4b56-bd12-bfb5c589c540)
![IMG_9519](https://github.com/peterantypas/maiana/assets/2204527/3045b7a4-2e1c-4bf2-b343-d0a9bf63674e)
![IMG_9517](https://github.com/peterantypas/maiana/assets/2204527/ae7928b1-a342-47f4-ad12-f4c6d280a923)
![IMG_9516](https://github.com/peterantypas/maiana/assets/2204527/228ddf72-e9e8-4238-ba6a-3e2c836e07d7)

**Reply by peterantypas** [reply]
This is very much in line with what I was thinking but never had time to do -- a harness that keeps the whole thing together. My only concern is the potential detuning effect on the GNSS antenna. Did you notice any effect on performance?

Also, how do the screws attach? Did you use threaded inserts? If you can post STL files, I'd love to print it and play with it (when I actually have time).

**Reply by dkroeske** [reply]
It's public in OnShape: https://cad.onshape.com/documents/557912cbeab422ef783f4b5b/w/3f95463100d65c3e7da1b912/e/dd57805c1886b47e0087a721
I will test it (and the performance of the GNSS antenna) in the next few weeks (indeed, available time is also my problem). The cable is fixed with two M4 nuts like this:
![image](https://github.com/peterantypas/maiana/assets/2204527/7a73ec61-fde4-4c20-bb26-220fd106399b)
